recipes/dbus/1.x.x/conanfile.py

```diff
@@ -0,0 +1,108 @@
+import os
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+class DbusConan(ConanFile):
+    name = "dbus"
+    license = "AFL-2.1"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.freedesktop.org/wiki/Software/dbus"
+    description = "D-Bus is a simple system for interprocess communication and coordination."
+    topics = ("conan", "dbus")
+    settings = "os", "compiler", "build_type", "arch"
+
+    options = {
+        "fPIC": [True, False],
+        "with_x11": [True, False],
+        "with_glib": [True, False],
+        "enable_assert": [True, False],
+        "enable_checks": [True, False],
+        "install_system_libs": [True, False]}
+
+    default_options = {
+        "fPIC": True,
+        "with_x11": False,
+        "with_glib": False,
+        "enable_assert": True,
+        "enable_checks": True,
+        "install_system_libs": False}
+
+    generators = "cmake", "cmake_find_package", "cmake_paths"
+
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+    _cmake = None
+
+    requires = (
+        "expat/2.2.9"
+    )
+
+    def config_options(self):
```

Comment:
```suggestion
    def configure(self):
```

Raise `ConanInvalidConfiguration` in `configure()`